### PR TITLE
[TEVA-2400] Make Steps Component More Responsive

### PR DESCRIPTION
**Pull request by josephhull676**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-2400

## Video thing

https://user-images.githubusercontent.com/30624173/114857375-170e9980-9de0-11eb-8c76-6f9f36a8bde9.mov

### CODE REVIEW

File: app/components/steps_component/steps_component.scss [hunk 1]
```diff
@@ -5,17 +5,15 @@
...
   @include govuk-media-query($from: desktop) {
     width: unset;
   }
 
   h2 {
     margin-bottom: govuk-spacing(2);
     margin-top: govuk-spacing(4);
+
+
   }
```
Comment (david-mears-dfe) [inline app/components/steps_component/steps_component.scss:17]: extra lines

Comment (alexbowen) [reply app/components/steps_component/steps_component.scss:17]: was literally writing the same. sass lint should have picked up on this, ill investigate why sepeartly

Comment (josephhull676) [reply app/components/steps_component/steps_component.scss:17]: Whoopsie

File: app/components/steps_component/steps_component.scss [hunk 2]
```diff
@@ -35,16 +31,25 @@
...
+      }
+
       &::before {
+        background-color: govuk-colour('white');
         border: 2px solid $govuk-border-colour;
         border-radius: 50%;
         content: counter(steps-counter);
         display: inline-block;
-        height: 100%;
+        left: 0;
```
Comment (alexbowen) [inline app/components/steps_component/steps_component.scss:44]: if this is for the position: absolute in mediaquery below i think it should go in that mediaquery?

**Comment by alexbowen** [comment]
tbh whilst this looks to work from your video, there are a few hardcoded measurements 52.5%, 35px and more that are a bit of a code smell to me that this could be done in a better way. if it works for now and needs merging, im willing to approve but i might have a tinker

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-3274.london.cloudapps.digital>

**Comment by csutter** [comment]
Just a thought, but: what if we didn't show the component at all on mobile? Since we don't let you interact with it anyway, it's really just a visual indication of your progress (and they still have a "step x out of y" in the heading anyway for that, and we could show the list of steps on the before you start page maybe).

It feels a bit odd to have it underneath the form, and it would waste a lot of space above the form...

**Review by david-mears-dfe — APPROVED** [review]
Approving even though I had a small hand in this

**Pull request merged**